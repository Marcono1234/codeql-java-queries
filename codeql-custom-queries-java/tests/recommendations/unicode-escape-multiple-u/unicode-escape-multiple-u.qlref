./recommendations/unicode-escape-multiple-u.ql
