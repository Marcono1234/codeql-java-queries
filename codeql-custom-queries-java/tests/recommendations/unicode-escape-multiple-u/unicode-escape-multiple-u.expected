| Test.java:2:4:2:10 | \\uu1234 | Contains Unicode escape `\\uu1234` which uses multiple `u`s |
| Test.java:11:9:11:17 | "\\uuABCD" | Contains Unicode escape `\\uuABCD` which uses multiple `u`s |
